wolfcrypt/src/asn.c

```diff
@@ -20051,6 +20053,17 @@ int PemToDer(const unsigned char* buff, long longSz, int type,
                         }
                     }
                 #endif /* !NO_DES3 */
+                #if !defined(NO_AES) && defined(HAVE_AES_CBC) && \
+                    defined(HAVE_AES_DECRYPT)
+                    if (info->cipherType == WC_CIPHER_AES_CBC) {
+                        if (der->length > AES_BLOCK_SIZE) {
+                            padVal = der->buffer[der->length-1];
+                            if (padVal <= AES_BLOCK_SIZE) {
```

Comment:
Then use `if (padVal <= der->length)`?

Reply:
The padding value must cannot be more than the block size if using PKCS #5/#7 padding.